pkg/daemon/ceph/client/osd.go

```diff
@@ -150,6 +154,22 @@ func OSDRemove(context *clusterd.Context, clusterName string, osdID int) (string
 	return string(buf), err
 }
 
+func OsdSafeToDestroy(context *clusterd.Context, clusterName string, osdID []string) (*SafeToDestroyStatus, error) {
```

Comment:
`ceph osd safe-to-destroy` can take multiple osd ids and return all the osds that are safe to destroy. So i took `osdID` as a `[]string`. So will it be fine if I take just the `osdID` as `int` or `string`, even though the output of the command after deserialization is a struct containing `[]int` ?

Reply:
Yes, sounds good to just add a single osd as input, but then you would need to handle the output as an array.